executor/executor.cc

```diff
@@ -732,6 +734,10 @@ void execute_one()
 		}
 	}
 
+#if SYZ_HAVE_CLOSE_FDS
+	close_fds();
```

Comment:
We don't write any coverage after this line, so close_fds() won't give us any additional coverage.
If this is supposed to give us more coverage, shouldn't we do this before we wait for unfinished syscalls and write coverage?

Reply:
We will write coverage after `close_fds()` when USB support is added.